metadata.txt

```diff
@@ -10,7 +10,7 @@
 name=Terra Antiqua
 qgisMinimumVersion=3.10
 description=This plugin creates digital elevation models of paleogeographic maps.
-version=1.0.3
+version=1.0.3-beta
```

Comment:
Version for this and other pull requests currently open should be 1.1-alfa/beta.

Reply:
### To keep it simple, we are NOT using alpha.
We use **_-beta_** and **_-rc_ (release candidate)** for the pre-release versions, such as these pull requests.
And (if needed) we use **_+com.xxx123_ (identifier of the commit)** or **_+is.123_ (number of the issue solved)** for internal builds